index.html

```diff
@@ -14,252 +14,400 @@
+                Experimental content creation feature that allows users to add
+                to an existing story over the course of a day without spamming
+                their friends.
+              </p>
+            </li>
+            <li>
+              <p class="daily-mobile">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+            </li>
+          </ul>
 
-        <h2 class="multi-Post">
-          Multi-Post<br />
-          Stories
-        </h2>
-        <div class="canopy-container">
-          <h2 class="canopy">CANOPY</h2>
-          <p class="date">• Back End Dev • 2015</p>
+          <div>
+            <button>html</button>
+            <button>css</button>
+            <button class="javascript">javaScript</button>
+          </div>
+          <button class="see-project">See Project</button>
         </div>
-        <p class="daily-sign">
-          A daily selection of privately<br />
-          personalized reads; no accounts or sign-ups required.
-        </p>
 
-        <ul class="button-container">
-          <li><button class="html">html</button></li>
-          <li><button class="css">css</button></li>
-          <li><button class="javascript">javaScript</button></li>
+        <ul class="images">
+          <li>
+            <img
+              src="img/Snapshoot Portfolio (3).png"
+              alt="card"
+              class="desktop-images"
+            />
+          </li>
+          <li>
+            <img
+              src="img/Snapshoot Portfolio (1).png"
+              alt="card"
+              class="mobile-images"
+            />
+          </li>
         </ul>
-
-        <a href="http://127.0.0.1:5500/index.html"
-          ><button class="btn">See Project</button></a
-        >
       </section>
 
-      <!-- multipost section -->
-      <section class="post-section">
-        <div class="postsection-image"></div>
-
-        <h2 class="tonic">Tonic</h2>
-        <div class="canopy-container">
-          <h2 class="canopy">CANOPY</h2>
-          <p class="date">• Back End Dev • 2015</p>
-        </div>
-        <p class="daily-sign">
-          A daily selection of privately<br />
-          personalized reads; no accounts or sign-ups required.
-        </p>
-
-        <ul class="button-container">
-          <li><button class="html">html</button></li>
-          <li><button class="css">css</button></li>
-          <li><button class="javascript">javaScript</button></li>
+      <section class="facebook-section">
+        <ul class="images">
+          <li>
+            <img
+              src="/img/Snapshoot Portfolio.png"
+              alt="card"
+              class="desktop-images"
+            />
+          </li>
+          <li>
+            <img
+              src="img/Snapshoot Portfolio (2).png"
+              alt="card"
+              class="mobile-images"
+            />
```

Comment:
- [ ] As the previous reviewer pointed, the image is still broken,  please consider addressing this issue. 

![image](https://user-images.githubusercontent.com/77038610/144577232-c667c332-99e7-4982-a858-50f59f033e78.png)